### PR TITLE
Editorial: use bikeshed shorthands for devicemotion firing steps

### PR DESCRIPTION
**Pull request by marcoscaceres**
Just bikeshed stuff... doesn't change the algorithm.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/deviceorientation/pull/176.html" title="Last updated on May 21, 2024, 12:43 AM UTC (29c2bb2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/deviceorientation/176/39c015e...29c2bb2.html" title="Last updated on May 21, 2024, 12:43 AM UTC (29c2bb2)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -540,65 +540,65 @@ The <dfn method for="DeviceMotionEvent">requestPermission()</dfn> method steps a
...
-      1. If <var>document</var> is not <a>allowed to use</a> the <a>policy-controlled feature</a> named <var>policy</var>, return.
-  1. Let <var>topLevelTraversable</var> be <var>window</var>'s <a for=Window>navigable</a>'s <a for=navigable>top-level traversable</a>.
-  1. Let <var>platformLinearAcceleration</var> be null.
-  1. If <var>topLevelTraversable</var>'s <a>virtual sensor mapping</a> <a for=map>contains</a> "<code>linear-acceleration</code>":
-      1. Let <var>virtualSensor</var> be <var>topLevelTraversable</var>'s <a>virtual sensor mapping</a>["<code>linear-acceleration</code>"].
-      1. If <var>virtualSensor</var>'s <a>can provide readings flag</a> is true, then set <var>platformLinearAcceleration</var> to the latest readings provided to <var>virtualSensor</var>.
+  1. Let |document| be |window|'s [=associated Document=].
+  1. If |document|'s [=Document/visibility state=] is not "<code>visible</code>", return.
+  1. [=list/For each=] |policy| of « "<a permission>"accelerometer"</a>", <a permission>"gyroscope"</a> »:
```
Comment (reillyeon) [inline index.bs:547]:
```suggestion
  1. [=list/For each=] |policy| of « <a permission>"accelerometer"</a>, <a permission>"gyroscope"</a> »:
```

File: index.bs [hunk 2]
```diff
@@ -540,65 +540,65 @@ The <dfn method for="DeviceMotionEvent">requestPermission()</dfn> method steps a
...
+    1. [=list/For each=] |permission| in « <a permission>"accelerometer"</a>, <a permission>"gyroscope"</a> »:
+        1. Let |state| be the result of <a>getting the current permission state</a> with |permission| and |environment|.
+        1. If |state| is not "{{PermissionState/granted}}", return.
+    1. [=Queue a global task=] on the [=device motion and orientation task source=] given |window| to run the following steps:
+        1. [=Fire an event=] named "devicemotion" at |window|, using {{DeviceMotionEvent}}, with the {{DeviceMotionEvent/acceleration}} attribute initialized to |acceleration|, the {{DeviceMotionEvent/accelerationIncludingGravity}} attribute initialized to |accelerationIncludingGravity|, the {{DeviceMotionEvent/rotationRate}} attribute initialized to |rotationRate|, and the {{DeviceMotionEvent/interval}} attribute initialized to |interval|.
```
Comment (reillyeon) [inline index.bs:601]: Link to #def-devicemotion?

File: index.bs [hunk 3]
```diff
@@ -540,65 +540,65 @@ The <dfn method for="DeviceMotionEvent">requestPermission()</dfn> method steps a
...
+  1. Let |platformRotationRate| be null.
+  1. If |topLevelTraversable|'s <a>virtual sensor mapping</a> <a for=map>contains</a> "<code>gyroscope</code>":
+      1. Let |virtualSensor| be |topLevelTraversable|'s <a>virtual sensor mapping</a>["<code>gyroscope</code>"].
+      1. If |virtualSensor|'s <a>can provide readings flag</a> is true, then set |platformRotationRate| to the latest readings provided to |virtualSensor|.
+  1. Otherwise, if the implementation is able to provide [=rotation rate=]:
+      1. Set |platformRotationRate| to the device's [=rotation rate=] about the X, Y and Z axes.
+  1. Let |rotationRate| be null.
+  1. If |platformRotationRate| is not null:
+      1. Set |rotationRate| to a <a>new</a> {{DeviceMotionEventRotationRate}} created in |window|'s <a for="global object">realm</a>.
+      1. Set |rotationRate|'s <a for="DeviceMotionEventRotationRate">x axis rotation rate</a> to |platformRotationRate|'s value about the X axis, or null if it cannot be provided.
```
Comment (reillyeon) [inline index.bs:589]: Use Bikeshed syntax for these references?

**Review by reillyeon — CHANGES_REQUESTED** [review]

File: index.bs [hunk 4]
```diff
@@ -540,65 +540,65 @@ The <dfn method for="DeviceMotionEvent">requestPermission()</dfn> method steps a
...
```
Comment (marcoscaceres) [inline index.bs:589]:
```suggestion
      1. Set |rotationRate|'s [= DeviceMotionEventRotationRate/x axis rotation rate=] to |platformRotationRate|'s value about the X axis, or null if it cannot be provided.
```

**Comment by marcoscaceres** [comment]
Should hopefully be good to go. Feel free to merge if it's all good @reillyeon 🙏

File: index.bs [hunk 5]
```diff
@@ -539,65 +539,65 @@ The <dfn method for="DeviceMotionEvent">requestPermission()</dfn> method steps a
...
+At an [=implementation-defined=] interval |interval|, the user agent must execute the following steps on a [=/navigable=]'s [=navigable/active window=] |window|:
+
+  1. Let |document| be |window|'s [=associated Document=].
+  1. If |document|'s [=Document/visibility state=] is not "`visible`", return.
+  1. [=list/For each=] |policy| of « <a permission>"accelerometer"</a>, <a permission>"gyroscope"</a> »:
+      1. If |document| is not [=allowed to use=] the [=policy-controlled feature=] named |policy|, return.
+  1. Let |topLevelTraversable| be |window|'s [=Window/navigable=]'s [=navigable/top-level traversable=].
+  1. Let |platformLinearAcceleration| be null.
+  1. If |topLevelTraversable|'s [=virtual sensor mapping=] [=map/contains=] "`linear`e>":
```
Comment (reillyeon) [inline index.bs:550]:
```suggestion
  1. If |topLevelTraversable|'s [=virtual sensor mapping=] [=map/contains=] "`linear-acceleration`":
```

File: index.bs [hunk 6]
```diff
@@ -539,65 +539,65 @@ The <dfn method for="DeviceMotionEvent">requestPermission()</dfn> method steps a
...
+      1. Let |virtualSensor| be |topLevelTraversable|'s [=virtual sensor mapping=]["`linear`e>"].
```
Comment (reillyeon) [inline index.bs:551]:
```suggestion
      1. Let |virtualSensor| be |topLevelTraversable|'s [=virtual sensor mapping=]["`linear-acceleration`"].
```

**Review by reillyeon — APPROVED** [review]

**Pull request merged**